src/SSHLibrary/library.py

```diff
@@ -12,11 +12,14 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
+from __future__ import print_function
 try:
     from robot.api import logger
 except ImportError:
     logger = None
 from robot.utils import ConnectionCache
+from robot.utils.robottypes import is_string, is_bytes
+from robot.utils.unic import unic
```

Comment:
Like I commented earlier, should import utils from the `robot.utils` package directly. Above three lines should thus be combined.